frontend/utils/getColorState.ts

```diff
@@ -0,0 +1,26 @@
+export type ColorState = "success" | "warn" | "error" | "info";
+
+export default (oee: number, isPaused: boolean): ColorState => {
+  if(isPaused) {
+    return "info";
+  } else if(oee > 80) {
+    return "success";
+  } else if(oee > 50) {
```

Comment:
Not a change for this PR, but in the future these limits will come from the API, and some machines may not have limits